specs/v1.yaml

```diff
@@ -422,6 +422,441 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks:
+    get:
+      summary: List Webhooks
+      description: List Webhooks for a subscription
```

Comment:
we don't expose a `subscription` externally today so this term is confusing and not necessary since we implicitly resolve the subscription based on their OAuth app metadata

Reply:
done